content/rs/references/rest-api/requests/ocsp/_index.md

```diff
@@ -0,0 +1,141 @@
+
+| Permission name |
+|-----------------|
+| [view_ocsp_config]({{<relref "/rs/references/rest-api/permissions#view_ocsp_config">}}) |
+
+### Request {#get-request} 
+
+#### Example HTTP request
+
+	GET /ocsp 
+
+
+#### Request headers
+
+| Key | Value | Description |
+|-----|-------|-------------|
+| Host | cnm.cluster.fqdn | Domain name |
+| Accept | application/json | Accepted media type |
+
+### Response {#get-response} 
+
+Returns an [OCSP configuration object]({{<relref "/rs/references/rest-api/objects/ocsp">}}).
+
+#### Example JSON body
+
+```json
+{
+    "ocsp_functionality": true,
+    "responder_url": "http://responder.ocsp.url.com",
+    "query_frequency": 3800,
+    "response_timeout": 2,
+    "recovery_frequency": 80,
+    "recovery_max_tries": 20
+}
+```
+
+### Error codes {#get-error-codes} 
+
+When errors are reported, the server may return a JSON object with `error_code` and `message` fields that provide additional information. The following are possible `error_code` values:
+
+| Code | Description |
+|------|-------------|
+| ocsp_unsupported_by_capability | Not all nodes support OCSP capability | 
+
+### Status codes {#get-status-codes} 
+
+| Code | Description |
+|------|-------------|
+| [200 OK](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.2.1) | Success |
+| [406 Not Acceptable](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.7) | Feature not supported in all nodes |
+
+## Update OCSP configuration {#put-ocsp}
+
+	PUT /v1/ocsp
+
+Updates the cluster's OCSP configuration.
+
+#### Required permissions
+
+| Permission name |
+|-----------------|
+| [config_ocsp]({{<relref "/rs/references/rest-api/permissions#config_ocsp">}}) |
+
+### Request {#put-request} 
+
+#### Example HTTP request
+
+	PUT /ocsp 
+
+#### Example JSON body
+
+```json
+{
+    "ocsp_functionality": true,
+    "query_frequency": 3800,
+    "response_timeout": 2,
+    "recovery_frequency": 80,
+    "recovery_max_tries": 20
+}
+```
+
+#### Request headers
+
+| Key | Value | Description |
+|-----|-------|-------------|
+| Host | cnm.cluster.fqdn | Domain name |
+| Accept | application/json | Accepted media type |
+
+#### Request body
+
+Include an [OCSP configuration object]({{<relref "/rs/references/rest-api/objects/ocsp">}}) with updated fields in the request body.
+
+### Response {#put-response} 
+
+Returns the updated [OCSP configuration object]({{<relref "/rs/references/rest-api/objects/ocsp">}}).
+
+### Error codes {#put-error-codes} 
+
+When errors are reported, the server may return a JSON object with `error_code` and `message` fields that provide additional information. The following are possible `error_code` values:
+
+| Code | Description |
+|------|-------------|
+| invalid_schema | An illegal parameter or a parameter with an illegal value |
+| no_responder_url | Tried to enable OCSP with no responder URL configured |
+| ocsp_unsupported_by_capability | Not all nodes support OCSP capability |
```

Comment:
Consider "OCSP capability not supported by all nodes". Also is `capability `in the message correct? Should it be `node` instead?